webots_ros2_epuck/launch/robot_launch.py

```diff
@@ -37,15 +37,15 @@ def generate_launch_description():
     ]
     webots = Node(
         package='webots_ros2_core',
-        node_executable='webots_launcher',
+        executable='webots_launcher',
```

Comment:
Just to make sure, is this compatible with eloquent?

Reply:
So, in `develop` branch we target Eloquent or we aim to support both, Foxy and Eloquent, at the same time.

Reply:
Ideally both. But if not possible we target eloquent and do the Foxy specific code in the Foxy branch.